src/cache/tracer_cache.jl

```diff
@@ -1,40 +1,27 @@
-using ClimaUtilities.ClimaArtifacts
 import Dates: Year
 import ClimaUtilities.TimeVaryingInputs:
     TimeVaryingInput, LinearPeriodFillingInterpolation
 
-function tracer_cache(
-    Y,
-    atmos,
-    prescribe_ozone,
-    prescribed_aerosol_names,
-    start_date,
-)
-    if isempty(prescribed_aerosol_names) && !prescribe_ozone
+ozone_cache(_, _, _) = (;)
+function ozone_cache(::PrescribedOzone, Y, start_date)
+    o3 = similar(Y.c.ρ)
+    prescribed_o3_timevaryinginput = TimeVaryingInput(
+        AA.ozone_concentration_file_path(; context = ClimaComms.context(Y.c)),
+        "vmro3",
+        axes(o3);
+        reference_date = start_date,
+        regridder_type = :InterpolationsRegridder,
+        method = LinearPeriodFillingInterpolation(Year(1)),
+    )
+    return (; o3, prescribed_o3_timevaryinginput)
+end
+
+function tracer_cache(Y, atmos, prescribed_aerosol_names, start_date)
+    if isempty(prescribed_aerosol_names)
```

Comment:
Here it will return an empty namedtuple if prescribe_ozone is true but there is no prescribed_aerosol?